README.md

```diff
@@ -257,6 +257,55 @@ networks: {
     },
   },
 ```
+## Upgrade contract
+1. Make sure that your deployments/[network] folder is empty/deleted
```

Comment:
Should we delete the deployements artifacts?

I think hardhat-deploy uses them when you are wanting to upgrade implementation. Check out his [issue](https://github.com/wighawag/hardhat-deploy/issues/70) on upgrading contracts owned by multisig